### PR TITLE
Dataset page

### PR DESCRIPTION
**Pull request by cameronbaney**
# Description

The purpose of this PR is to add the dataset page.

127 0 0 1_5000_browse_(Laptop with HiDPI screen)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Navigate directly to [a dataset's page](http://127.0.0.1:5000/browse/#/datasets/4)
- Should load the dataset, and display all its data.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by cameronbaney** [comment]
I am removing some unused code, and adding an error state.

File: dat_core/vue_app/src/mixins/logger/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+     * @param {String} fnName method to use, i.e., warn, log, error, trace, etc.
+     * @param {Boolean} disabled turns logging off completely
+     * @returns {Object} console function or no-op
+     */
+    logger: function (fnArgs, fnName = 'log', disabled = false) {
+      if (disabled) {
+        return
+      }
+      const msg = Array.isArray(fnArgs) ? fnArgs : [fnArgs]
+      const isProduction = location.href.indexOf('app.blackfynn.') >= 0
```
Comment (clohr) [inline dat_core/vue_app/src/mixins/logger/index.js]: We'll need to update this domain to match the data portal's.

File: dat_core/vue_app/src/mixins/request/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    handleXhrError: function (err) {
+      this.isLoading = false
+      const status = prop('status', err)
+      // logout
+      if (status === 401) {
+        return this.handleLogout()
+      }
+      // unauthorized
+      if (status === 403) {
+        console.error('unauthorized')
```
Comment (clohr) [inline dat_core/vue_app/src/mixins/request/index.js]: We can use the `logger` here.

**Comment by cameronbaney** [comment]
I have fixed an issue with the broken dataset banner not loading, and removed a lot of unused icons.

File: dat_core/vue_app/src/mixins/request/index.js (hunk 2 above)
Comment (cameronbaney) [reply dat_core/vue_app/src/mixins/request/index.js]: I changed this to set an `errorLoading` flag that we can do what we want with down the road.

File: dat_core/vue_app/src/mixins/logger/index.js (hunk 1 above)
Comment (cameronbaney) [reply dat_core/vue_app/src/mixins/logger/index.js]: Good point. What is that URL going to be?

Comment (clohr) [reply dat_core/vue_app/src/mixins/logger/index.js]: https://data.sparc.science

Comment (cameronbaney) [reply dat_core/vue_app/src/mixins/logger/index.js]: Awesome, thank you. I have changed the URL in the logger.

**Pull request merged**